Source/control.cpp

```diff
@@ -1065,6 +1065,14 @@ void DoSpeedBook()
 	SetCursorPos({ x, y });
 }
 
+void ClearPanBtn()

```

Comment:
Seams like this could replace line 936-939 in `InitControlPan()`, it's a bit unclear what the state of `drawbtnflag` would be at that point, probably `true`, but I doubt it matters.

Reply:
Thanks, I forgot about the build in virtual mouse events in SDL :)

Reply:
The call stack there is `ShowProgress()` -> `LoadGame()` -> `LoadGameLevel()` -> `InitControlPan()`. In other words, it's called while the load screen with the progress bar is showing so `drawbtnflag` should indeed be irrelevant.